File: rxcv4j/src/main/java/com/cv4j/rxjava/WrappedCV4JImage.java
package com.cv4j.rxjava;

import com.cv4j.core.datamodel.CV4JImage;
import com.cv4j.core.filters.CommonFilter;

import java.util.List;

/**
 * Created by Tony Shen on 2017/3/31.
 */

public class WrappedCV4JImage {

    public CV4JImage image;
    public List<CommonFilter> filters;

    public WrappedCV4JImage(CV4JImage image,List<CommonFilter> filters) {

        this.image = image;
        this.filters = filters;
    }
}

File: rxcv4j/src/main/java/com/cv4j/rxjava/WrappedCV4JImage.kt
package com.cv4j.rxjava

import com.cv4j.core.datamodel.CV4JImage
import com.cv4j.core.filters.CommonFilter

/**
 * Created by Tony Shen on 2017/5/19.
 */
data class WrappedCV4JImage(val image: CV4JImage, val filters: List<CommonFilter>) {
}